literature.bib

```diff
@@ -3293,6 +3385,25 @@ @article{RTS+:JLAMP16
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2015 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PyEDA15,
+	author 		= {Chris Drake},
+	title  		= {{PyEDA: Data Structures and Algorithms for Electronic Design Automation}},
+	booktitle 	= {{Proc.\ Conf.\ on Python in Science (SciPy)}},
```

Comment:
Is there a reasony why this is not a variable in MyAbrv?

Reply:
Single occurrence, doubtful there will be more.